consensus/types/src/beacon_state/pubkey_cache.rs

```diff
@@ -1,7 +1,7 @@
 use crate::*;
 use rpds::HashTrieMapSync as HashTrieMap;
 
-type ValidatorIndex = usize;
+pub type ValidatorIndex = usize;
```

Comment:
this is a type alias, if `consensus` is exposing a function/method that requires `ValidatorIndex` we should probably either switch it to `usize` or make `ValidatorIndex` a [`newtype`](https://doc.rust-lang.org/rust-by-example/generics/new_types.html) to avoid breaking change issues

Reply:
Hey thanks for this! I started changing the `ValidatorIndex` to a newtype but it ended up requiring too many unexpected downstream changes for me to justify just the re-export. I would like to keep it a newtype in anchor so I believe it is best to just define it there, hence why I closed this pr.